src/server/browserContext.ts

```diff
@@ -258,7 +258,7 @@ export abstract class BrowserContext extends SdkObject {
       // Collect videos/downloads that we will await.
       const promises: Promise<any>[] = [];
       for (const download of this._downloads)
-        promises.push(download.delete());
+        promises.push(download.forceDelete());
```

Comment:
I believe we should call rm only after the context has been closed as otherwise it may file because the file is still held by the browser or am I missing something?

Reply:
Good catch!